src/main/java/edu/kpi/testcourse/rest/UrlController.java

```diff
@@ -35,6 +35,9 @@
   @Inject
   AuthorizationMockServiceImpl authorizationMockService;
 
+  @Inject
+  JsonToolJacksonImpl jsonTool;
```

Comment:
You should inject JsonTool, JsonToolJacksonImpl will be chosen as only implementation now existent, or you can configure it in BeanFactory.
```suggestion
  JsonTool jsonTool;
```

Reply:
Thanks for the good note! Checked your and Vadim's code - will fix my part of the project.